src/Doctrine/EntityRegenerator.php

```diff
@@ -38,7 +38,7 @@ public function regenerateEntities(string $classOrNamespace): void
     {
         try {
             $metadata = $this->doctrineHelper->getMetadata($classOrNamespace);
-        } catch (MappingException|LegacyCommonMappingException|PersistenceMappingException $mappingException) {
+        } catch (MappingException|LegacyCommonMappingException|PersistenceMappingException) {
```

Comment:
`Doctrine\Common` namespaced imports e.g. `LegacyCommonMappingException` will be removed in a separate PR as there are a lot of changes related to outdated legacy classes that need to be made based on a git grep